templates/repo/issue/fields/textarea.tmpl

```diff
@@ -1,6 +1,21 @@
-<div class="field">
+<div class="field textarea">
 	{{template "repo/issue/fields/header" .}}
 	{{/* FIXME: preview markdown result */}}
-	{{/* FIXME: required validation for markdown editor */}}
-	<textarea name="form-field-{{.item.ID}}" placeholder="{{.item.Attributes.placeholder}}" {{if and .item.Validations.required .item.Attributes.render}}required{{end}}>{{.item.Attributes.value}}</textarea>
+	<textarea class="fake" name="form-field-{{.item.ID}}" placeholder="{{.item.Attributes.placeholder}}" {{if and .item.Validations.required .item.Attributes.render}}required{{end}}>{{.item.Attributes.value}}</textarea>
+
+	{{template "shared/combomarkdowneditor" (dict
+		"locale" .Locale
+		"MarkdownPreviewUrl" (print .RepoLink "/markup")
+		"MarkdownPreviewContext" .RepoLink
+		"TextareaName" "content"
+		"TextareaContent" .item.Attributes.value
+		"TextareaPlaceholder"  .item.Attributes.placeholder
+		"DropzoneParentContainer" "form, .ui.form"
```

Comment:
Maybe `DropzoneParentContainer` is incorrect here, have no idea about how to use this now. 
Need to read the js code later.